crates/core_arch/src/wasm32/simd128.rs

```diff
@@ -499,7 +499,7 @@ pub unsafe fn v128_store64_lane<const L: usize>(v: v128, m: *mut u64) {
         a15 = 15,
     )
 )]
-pub const unsafe fn v128_const(
+pub const unsafe fn i8x16_const(
```

Comment:
The `const` suffix here feels a bit strange since the intrinsic works with runtime values as well. Have you considered just removing the suffix and naming the function just `i8x16`? That way it looks somewhat like a constructor in the code.

Reply:
How about `i8x16_new`? I'm a bit hesitant to take the small name of `i8x16`, but only slightly, I could imagine it working as well.

Reply:
I'd say try `i8x16` at first and then revisit this question when stabilizing to see how it worked out.